index.yaml

```diff
@@ -1110,35 +1074,7 @@ entries:
       catalog.cattle.io/ui-component: rancher-external-ip-webhook
     apiVersion: v1
     appVersion: v0.1.6
-    created: "2021-06-16T21:05:17.243534-07:00"
-    description: |
-      Deploy the external-ip-webhook to mitigate k8s CVE-2020-8554
-    digest: 72bcf1b9309ff0db9db04af7c5c712deeecab1b0d13d2b541abd8bd68f64edcc
-    home: https://github.com/rancher/externalip-webhook
-    keywords:
-    - cve
-    - externalip
-    - webhook
-    - security
-    maintainers:
-    - email: raul@rancher.com
-      name: rawmind0
-    name: rancher-external-ip-webhook
-    sources:
-    - https://github.com/rancher/externalip-webhook
-    urls:
-    - assets/rancher-external-ip-webhook/rancher-external-ip-webhook-100.0.0+up0.1.6.tgz
-    version: 100.0.0+up0.1.6
```

Comment:
100.0.0+up0.1.6 is deleted because we delete the package dir of the chart. 
the rest versions are pulled from the release-v2.6 branch.